assets/admission-webhook/network-policy-downstream.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  annotations:
+    include.release.openshift.io/hypershift: "true"
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
```

Comment:
(non-blocking) I believe that `include.release.openshift.io/*` annotations are only relevant for manifests deployed by CVO. Do we need them here?